src/features/codeLensProvider.ts

```diff
@@ -5,15 +5,17 @@
 
 'use strict';
 
-import { OmniSharpServer } from '../omnisharp/server';
-import TelemetryReporter from 'vscode-extension-telemetry';
-import TestManager from './dotnetTest';
-import * as vscode from 'vscode';
-import { toRange, toLocation } from '../omnisharp/typeConvertion';
-import AbstractProvider from './abstractProvider';
 import * as protocol from '../omnisharp/protocol';
 import * as serverUtils from '../omnisharp/utils';
+import * as vscode from 'vscode';
+
+import { toLocation, toRange } from '../omnisharp/typeConvertion';
```

Comment:
What's the sorting order here? Alphabetical by file path?

Reply:
I use an extension called `Sort Typescript Imports`. It's fairly configurable, so we can decide on a sort order that we're happiest with. By default:

`sorts by the type and name of the import. Namespace imports are first, followed by default imports, named imports, and unnamed imports.`